app/src/main/res/layout/subscription_item.xml

```diff
@@ -37,12 +36,20 @@
         android:id="@+id/triangleCountView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
         android:layout_alignParentTop="true"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
         app:backgroundColor="?attr/colorSecondary"
         app:corner="rightTop"
         app:primaryText="Test"
         app:primaryTextColor="?attr/colorOnSecondary"
         app:primaryTextSize="12sp" />
+
+    
+
+    <CheckBox
+        android:id="@+id/selectCheckBox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:buttonTint="#FFFFFF" />
```

Comment:
What about white covers? I would do something like the mock-up by @keunes in https://github.com/AntennaPod/AntennaPod/issues/2788#issuecomment-611777506: Use a blue circle with a white checkmark. That makes it independent of the colors of the image. I wouldn't explicitly show the plus icon, though.

Reply:
Hello
I like the selector indicators, but they're not always too visible. I would make some slight changes:
* Give them a white-wash filling when not selected
* Make them brand blue with white checkmark when selected
That would look a bit like this:
![Gallery selection](https://user-images.githubusercontent.com/11229646/126892048-097a9cc2-0c06-46e6-8533-ac308f359afe.jpg)

If at all possible, I would also not hard-code the colour, but use reference to the central colour list. The brand blue should be defined already with the swipe actions, and there also is a white/grey wash defined already.

Reply:
> What about white covers? I would do something like the mock-up by @keunes in [#2788 (comment)](https://github.com/AntennaPod/AntennaPod/issues/2788#issuecomment-611777506): Use a blue circle with a white checkmark. That makes it independent of the colors of the image. I wouldn't explicitly show the plus icon, though.

I like the idea, but I figured since we already use a checkbox for episode mult select, we should be consistent and use it for subscription multi select also. I really like @keunes idea of using a white-washed checkbox to make it clearer. 
I also had an idea to use a blue checkbox with a translucent white background when selected. That way it will be more clear when an item is selected. What do you think @keunes?